course3/homework-string/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_FLAGS "-g -Wall")
```

Comment:
Don't set '-g' in your CMake.
To choose whether you want to build in debug/release or any the of the available modes, use -DCMAKE_BUILD_TYPE=XXX when running cmake for the first time.
This way you can have multiple build directories with different configuration.

You only need to set the var on the first run, it will be cached for subsequent runs